internal/provider/kubernetes/status.go

```diff
@@ -568,8 +568,7 @@ func (r *gatewayAPIReconciler) updateStatusForGateway(ctx context.Context, gtw *
 		r.log.Info("failed to get Service for gateway",
 			"namespace", gtw.Namespace, "name", gtw.Name)
 	}
-	// update accepted condition
-	status.UpdateGatewayStatusAcceptedCondition(gtw, true)
```

Comment:
curious why this was removed

Reply:
I think we should respect the logic in `controller.go`.

Reply:
this made gateway always accepted, even the ref EnvoyProxy is not found.